### ISSUE TITLE
Cross-eye stereo option

### ISSUE THREAD

**Issue by basveeling** [maartenbreddels/ipyvolume#24]
The stereo viewing options is great, but requires a VR device. By adding an option to switch the left and right eye images, stereo 3d can be achieved with the cross-eyed[1] technique.

- [1] https://en.wikipedia.org/wiki/Stereoscopy#Freeviewing

On a personal note, thanks for the great talk at PyData Amsterdam, Maarten!

**Comment by maartenbreddels** [comment]
Good idea, I've received the suggestion before, but didn't understand exactly what the idea was, but the link helps. I'll take this into consideration when redoing some of the default controls.

And thanks, the positive feedback is good for the motivation :)

**Comment by maartenbreddels** [comment]
Implemented in 2ea543dd600eeaf899487d30633182b6ad04cf08
To try out:
```python
pylab.figure(controls_vr=True)
pylab.examples.klein_bottle()
```
This attaches a slider to `Figure.eye_separation`
